cypress/support/commands.ts

```diff
@@ -15,14 +15,27 @@ export const signin = (): Cypress.Chainable<Cypress.Response> => {
         })
     })
   \*/
-
   return cy
     .setupUser()
     .then(() => cy.visit('/api/v2/signin'))
     .then(() => cy.get('#login').type(Cypress.env('username')))
     .then(() => cy.get('#password').type(Cypress.env('password')))
     .then(() => cy.get('#submit-login').click())
-    .then(() => cy.get('.theme-btn--success').click())
+    .then(() => {
+      cy.get('body').then($body => {
+        /**
```

Comment:
i think this'll get cleaned up when we start testing the actual signin page and not making concessions for dex